tools-v2/README.md

```diff
@@ -49,32 +49,46 @@ A tool for CurveFS & CurveBs.
         - [list dir](#list-dir)
         - [list space](#list-space)
         - [list chunkserver](#list-chunkserver)
+        - [list scan-status](#list-scan-status)
     - [clean-recycle](#clean-recycle)
     - [query](#query-1)
         - [query file](#query-file)
         - [query chunk](#query-chunk)
         - [query segment](#query-segment)
+        - [query scan-status](#query-scan-status)
     - [status](#status-1)
       - [status etcd](#status-etcd-1)
       - [status mds](#status-mds-1)
       - [status client](#status-client)
       - [status snapshotserver](#status-snapshotserver)
       - [status chunkserver](#status-chunkserver)
+      - [status etcd](#status-etcd-1)
+      - [status mds](#status-mds-1)
```

Comment:
repeat.
There are many mistakes in this reamdme, please correct them.

Reply:
ok